requirements.txt

```diff
@@ -17,4 +17,6 @@ pytest==3.5.0
 pytest-cov==2.5.1
 six==1.11.0
 stix-pattern-translator==0.1.0
+stix2-patterns==0.6.0
```

Comment:
general question... why don't we have requirements-dev.txt in this project?

Reply:
We didn't really get that far. The standard convention is to just have a requirements.txt. Travis actually looks for requirements.txt by default so in our private repo's we've configured it to use requirements-dev instead.

We should probably make one though